tests/app/submitter/conftest.py

```diff
@@ -65,12 +67,86 @@ def parse_metadata(claims, schema_metadata):
     return metadata
 
 
+@pytest.fixture
+def fake_metadata_v2():
+    def parse_metadata_v2(claims, schema_metadata):
+        runner_claims = validate_runner_claims_v2(claims)
+        questionnaire_claims = validate_questionnaire_claims(
+            claims["survey_metadata"]["data"], schema_metadata
+        )
+
+        for key, value in questionnaire_claims.items():
+            runner_claims["survey_metadata"]["data"][key] = value
+
+        return runner_claims
```

Comment:
Why does this fixture have all of this in?.

Reply:
Removed now